components/verify/Verifier.tsx

```diff
@@ -101,6 +101,16 @@ const Verifier: React.FC<VerifierProps> = ({ wrappedDocument }) => {
         };
       }
 
+      if (document.$template.url.slice(-6) === "gov.sg") {
```

Comment:
May I suggest to check against the issuer domain (i.e. `issuers[x].identityProof.location`) instead of the template URL?

```json
"issuers": [
  {
    "id": "d0ecffb7-7cf3-4439-a80d-d118ecee599b:string:did:ethr:0xE39479928Cc4EfFE50774488780B9f616bd4B830",
    "revocation": {
      "type": "9622679e-ccf1-4c0c-afee-391e473324c5:string:NONE"
    },
    "name": "43cb32ce-d9e1-4ac1-b876-081da8d95887:string:SAMPLE CLINIC",
    "identityProof": {
      "type": "cf09bc1c-55ae-465f-9aa3-7a9c7e121257:string:DNS-DID",
      "location": "5d81d5c7-7b2a-423a-89b1-63d4ad928357:string:donotverify.testing.verify.gov.sg",
      "key": "c355486e-8215-4153-a51c-2b068a1c7ccc:string:did:ethr:0xE39479928Cc4EfFE50774488780B9f616bd4B830#controller"
    }
  }
],
```

Reply:
There's also a few utilities from `oa-verify` like `getIssuerIdentityFragments()` that can help you extract out the issuer's domain. Might be useful for this case?

https://github.com/Open-Attestation/oa-verify#list-of-utilities

Reply:
Amended to check against the issuer domain which is already a state variable.

Reply:
Maybe we can use `endsWith` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) instead of `slice`?

Reply:
Agreed! I think can also reference this: https://github.com/Open-Attestation/verify.gov.sg/blob/6e9020c431759443624b783d44cfca39e7141396/utils/notarise-healthcerts.ts#L12-L18

Reply:
Okay! Changed to use `endsWith` instead